test-rpc-functions/src/rpc.rs

```diff
@@ -97,7 +99,8 @@ trait RpcTestFunctionsRpc {
 #[async_trait::async_trait]
 impl RpcTestFunctionsRpcServer for super::RpcTestFunctionsHandle {
     async fn genesis_pool_id(&self) -> rpc::Result<Option<String>> {
-        let (genesis_pool_id, genesis_stake_pool_data, _, _, _, _) = regtest_genesis_values();
+        let (genesis_pool_id, genesis_stake_pool_data, _, _, _, _) =
+            genesis_values(GenesisStakingSettings::new(""));
```

Comment:
If this is the default value, it's preferable to use the Default::default() trait function to indicate what this really is.

Reply:
Fixed.